src/Tribe/Main.php

```diff
@@ -256,48 +255,20 @@ public function hooks() {
 	/**
 	 * Used to add our beloved tickets to the JSON-LD markup
 	 *
+	 * @deprecated
+	 *
 	 * @param  array   $data The actual json-ld variable
 	 * @param  array   $args Arguments used to create the Markup
 	 * @param  WP_Post $post What post does this referer too
-	 * @return array
+	 * @return false
 	 */
 	public function inject_tickets_json_ld( $data, $args, $post ) {
-		if ( 'Event' !== $data->{'@type'} ) {
-			return $data;
-		}
-
-		$tickets = Tribe__Tickets__Tickets::get_all_event_tickets( $post->ID );
-
-		if ( empty( $tickets ) ) {
-			return $data;
-		}
-
-		$data->offers = array();
-		$startdates = array();
-		$enddates = array();
-		$in_stock = array();
-		foreach ( $tickets as $ticket ) {
-			$in_stock[] = $ticket->is_in_stock();
-
-			if ( $ticket->is_in_stock() ) {
-				$startdates[] = $ticket->start_date();
-				$enddates[] = $ticket->end_date();
-			}
-		}
-
-		$offer = (object) array(
-			'@type' => 'AggregateOffer',
-			'price' => tribe_get_cost( $post->ID ),
-
-			// Do not translate these strings
-			'availability' => 'http://schema.org/' . in_array( true, $in_stock ) ? 'InStock' : 'OutOfStock',
-			'availabilityStarts' => get_gmt_from_date( date( Tribe__Date_Utils::DBDATETIMEFORMAT, min( $startdates ) ), 'c' ),
-			'availabilityEnds' => get_gmt_from_date( date( Tribe__Date_Utils::DBDATETIMEFORMAT, max( $enddates ) ), 'c' ),
-		);
-
-		$data->offers = $offer;
+		/**
+		 * @todo remove this after 4.4
+		 */
+		_deprecated_function( __CLASS__ . '::' . __METHOD__, '4.2', 'Tribe__Tickets__JSON_LD__Order' );
```

Comment:
The first parameter would become: `Tribe__Event__Tickets::Tribe__Event__Tickets::inject_tickets_json_ld`

Using `__METHOD__` all by itself should do what you are looking to do.